screens/HomeScreen.tsx

```diff
@@ -223,6 +233,28 @@ const HomeScreen = ({
     return tokensSorted;
   }, [balances.slpTokens, tokensById]);
 
+  const blacklistTokenData: WalletSection["data"] = useMemo(() => {
+    let array: WalletSection["data"] = [];
+
+    for (const each of tokenData) {
+      if (tokenBlacklist.includes(each.tokenId)) {
+        array = [...array, each];
+      }
+    }
+    return array;
+  }, [tokenData]);
```

Comment:
This `memo` value relies on `tokenBlacklist`, so that must be added as a dependency here, same with the curatedTokenData below.